src/components/themes/blueradiance/theme.css

```diff
@@ -471,3 +464,7 @@ html {
 .metadataSidebarIcon {
     color: #00a4dc
 }
+
+.detailPagePrimaryContainer {
+    background-color: rgba(0, 0, 0, 0.4)
```

Comment:
I feel like this value should just be a default so it isn't required in all the themes.

Reply:
the reason i did that to match the theme background 

currently
![Annotation 2020-01-01 193648](https://user-images.githubusercontent.com/32230989/71643786-c9dfe480-2cce-11ea-9902-1972c2571740.jpg)
![Annotation 2020-01-01 1936482](https://user-images.githubusercontent.com/32230989/71643787-ca787b00-2cce-11ea-9c5f-cc4bebd94f6e.jpg)

if we set form the default 
![Annotation 2020-01-01 19364823](https://user-images.githubusercontent.com/32230989/71643809-090e3580-2ccf-11ea-9740-6580ee6549c3.jpg)

Reply:
change style  to blurred i think this is glass effect you said on matrix 
https://github.com/jellyfin/jellyfin-web/pull/611/commits/ac9fece8cca15d93daa28afb4ccf09a6a3387d0b